doc/website/getting-started/installation.md

```diff
@@ -4,6 +4,19 @@ iceoryx_utils and iceoryx_posh are deployed as independent cmake packages. Posh
 
 ## Prerequisites
 
+### General
+
+#### Cyclone DDS
```

Comment:
It is somehow confusing to start right away with installing dependencies for cyclonedds which are not needed for a basic iceoryx build. I would move that section below the `Linux` chapter here.

Additionally the chapter `General` is introduced with only `Cyclone DDS` in it. Is this on purpose?

Reply:
I agree with @dkroenke. These are not the most basic installation steps. Please move this to `doc/website/advanced/installation-guide-for-contributors.md`

Reply:
@mossmaurice Please take a look at #525 . This document is the first you read if you would like to use iceoryx with all the official supported features, like DDS. 

A user was missing the DDS part, therefore I added it.

Reply:
You're correct. Ignore my comment above.